src/GovUk.Education.ExploreEducationStatistics.Content.Api/Controllers/MetaGuidanceController.cs

```diff
@@ -17,11 +17,25 @@ public MetaGuidanceController(IMetaGuidanceService metaGuidanceService)
             _metaGuidanceService = metaGuidanceService;
         }
 
-        [HttpGet("publications/{publicationSlug}/release/{releaseSlug}/meta-guidance")]
-        public async Task<ActionResult<MetaGuidanceViewModel>> Get(string publicationSlug,
+        [HttpGet("publications/{publicationSlug}/releases/latest/meta-guidance")]
+        public async Task<ActionResult<MetaGuidanceViewModel>> GetLatest(string publicationSlug)
+        {
+            return await _metaGuidanceService.Get(
+                    PublicContentPublicationPath(publicationSlug),
+                    PublicContentLatestReleasePath(publicationSlug)
+                )
+                .HandleFailuresOrOk();
+        }
+
+        [HttpGet("publications/{publicationSlug}/releases/{releaseSlug}/meta-guidance")]
+        public async Task<ActionResult<MetaGuidanceViewModel>> Get(
+            string publicationSlug,
             string releaseSlug)
         {
-            return await _metaGuidanceService.Get(PublicContentReleasePath(publicationSlug, releaseSlug))
+            return await _metaGuidanceService.Get(
+                    PublicContentReleasePath(publicationSlug, releaseSlug),
```

Comment:
Publication and Release path params are in the incorrect order and you end up getting this

![image](https://user-images.githubusercontent.com/4147126/96974741-8c746e00-1511-11eb-9e41-4a0746907cd9.png)

Reply:
Great catch, I've also used named parameters to make it more obvious as well now.